worlds/sm64ex/Rules.py

```diff
@@ -40,30 +63,35 @@ def set_rules(world, player: int, area_connections: dict):
             fix_reg(randomized_entrances, SM64Levels.CAVERN_OF_THE_METAL_CAP, {"Hazy Maze Cave"}, swapdict, world)
 
     # Destination Format: LVL | AREA with LVL = LEVEL_x, AREA = Area as used in sm64 code
-    # Cast to int to not rely on availability of SM64Levels enum. Will cause crash in MultiServer otherwise
```

Comment:
You need to restore this comment

Reply:
Nevermind, it looks like I could have undid the force push from the command line, but I did it from PyCharm.  Comment is back.  How do I give you and https://github.com/RBmans credit, again?  He assisted with Rules.py.

Reply:
`Co-authored-by: name <additional-dev-1@example.com>` in the commit message for RBman. You can get the mail from his original commit.

I only did minor things so no need